index.bs

```diff
@@ -2894,14 +2911,17 @@ Given an [=attribution rate-limit record=] |newRecord|:
 <h3 dfn id="should-block-attribution-for-rate-limits">Should attribution be blocked by rate limits</h3>
 
 Given an [=attribution trigger=] |trigger|, an [=attribution source=]
-|sourceToAttribute|, and an [=attribution rate-limit record=] |newRecord|:
-
-1. If the result of running [=should attribution be blocked by attribution rate limit=] with |trigger| and
-    |sourceToAttribute| is <strong>blocked</strong>:
-    1. Let |debugData| be the result of running [=obtain debug data on trigger registration=]
-        with "<code>[=trigger debug data type/trigger-attributions-per-source-destination-limit=]</code>", |trigger|, |sourceToAttribute| and
+|sourceToAttribute|, and an [=attribution rate-limit record/scope=] |scope|:
+
+1. If the result of running [=should attribution be blocked by attribution rate limit=] with |trigger|, |sourceToAttribute|
+    and |scope| is <strong>blocked</strong>:
+    1. Let |debugDataType| be "<code>[=trigger debug data type/trigger-event-attributions-per-source-destination-limit=]</code>"
+    2. If |scope| is "<code>[=rate-limit scope/aggregatable-attribution=]</code>", set |debugDataType| to 
+        "<code>[=trigger debug data type/trigger-aggregatable-attributions-per-source-destination-limit=]</code>"
+    3. Let |debugData| be the result of running [=obtain debug data on trigger registration=]
+        with |debugDataType|, |trigger|, |sourceToAttribute| and
         [=obtain debug data on trigger registration/report=] set to null.
-    1. Return the [=triggering result=] ("<code>[=triggering status/dropped=]</code>", |debugData|).
+    4. Return the [=triggering result=] ("<code>[=triggering status/dropped=]</code>", |debugData|).
```

Comment:
We use `1.` for numbering each step to avoid having to renumber when steps are added or deleted.